remoteworkitem/tracker.go

```diff
@@ -5,7 +5,7 @@ import "github.com/almighty/almighty-core/models"
 // Tracker represents tracker configuration
 type Tracker struct {
 	models.Lifecycle
-	ID uint64 `gorm:"primary_key"`
+	ID string `gorm:"primary_key"`
```

Comment:
@baijum you are right, it will not work, working around it, will have to change it to `int` again i think
@aslakknutsen thoughts